endpoints.json

```diff
@@ -75,6 +75,20 @@
       "votes": 0,
       "created_at": "2024-05-30T12:52:35.697Z"
     }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "responds with article with new vote count",
+    "queries": [],
+    "exampleResponse":   {
```

Comment:
Dont forget to include the object body we expect to be sent with this endpoint. 
`{
       inc_votes: 27
}`